python/semantic_kernel/agents/open_ai/azure_assistant_agent.py

```diff
@@ -275,33 +279,44 @@ async def create(
 
         assistant_create_kwargs: dict[str, Any] = {}
 
+        code_interpreter_file_ids_combined: list[str] = []
+
+        if code_interpreter_file_ids is not None:
```

Comment:
There is some similar code between the Azure Assistant and the OpenAI Assistant that I will refactor in a later iteration.